test/e2e_node/mirror_pod_grace_period_test.go

```diff
@@ -56,61 +53,36 @@ var _ = SIGDescribe("MirrorPodWithGracePeriod", func() {
 
 		ginkgo.It("mirror pod termination should satisfy grace period when static pod is deleted [NodeConformance]", func() {
 			ginkgo.By("get mirror pod uid")
-			pod, err := f.ClientSet.CoreV1().Pods(ns).Get(context.TODO(), mirrorPodName, metav1.GetOptions{})
+			_, err := f.ClientSet.CoreV1().Pods(ns).Get(context.TODO(), mirrorPodName, metav1.GetOptions{})
 			framework.ExpectNoError(err)
-			uid := pod.UID
+
+			start := time.Now()
 
 			ginkgo.By("delete the static pod")
 			file := staticPodPath(podPath, staticPodName, ns)
 			framework.Logf("deleting static pod manifest %q", file)
 			err = os.Remove(file)
 			framework.ExpectNoError(err)
 
-			ginkgo.By("wait for the mirror pod to be running for grace period")
-			gomega.Consistently(func() error {
-				return checkMirrorPodRunningWithUID(f.ClientSet, mirrorPodName, ns, uid)
-			}, 19*time.Second, 200*time.Millisecond).Should(gomega.BeNil())
-		})
-
-		ginkgo.It("mirror pod termination should satisfy grace period when static pod is updated [NodeConformance]", func() {
```

Comment:
I guess the e2e test of this PR demonstrates the failed case. 
or you can reproduce it using https://github.com/kubernetes/kubernetes/issues/97722#issue-778928060 original issue's description.